src/main/java/com/hyperwallet/clientsdk/model/HyperwalletClientToken.java

```diff
@@ -0,0 +1,10 @@
+package com.hyperwallet.clientsdk.model;
+
+public class HyperwalletClientToken {
+
+    private String value;
+
+    public String getValue() {
+        return value;
+    }
```

Comment:
missing setter

Reply:
right. but I think we don't need setter in this case.